work2/Negtive25/Bonus/ArrayPrint/ThreadToPrint.java

```diff
@@ -0,0 +1,11 @@
+public class ThreadToPrint extends Thread {
+    int[]arr;
```

Comment:
`int[]arr;`
Do not use default decorate field
currect is: 
`private int[] arr;`